dyn_em/module_initialize_real.F

```diff
@@ -3060,7 +3060,7 @@ SUBROUTINE init_domain_rk ( grid &
 
       !  Split NUDAPT  Urban Parameters
 
-      IF ( ( config_flags%sf_urban_physics == 1 ) .OR. ( config_flags%sf_urban_physics == 2 ) .OR. ( config_flags%sf_urban_physics == 3 ) ) THEN
+      IF ( ( config_flags%sf_urban_physics == 1 ) .OR. ( config_flags%sf_urban_physics == 2 ) .OR. ( config_flags%sf_urban_physics == 3 ) .OR. ( config_flags%sf_urban_physics == 4 ) ) THEN
```

Comment:
@dmey 
Break this IF test into 2 lines, something like:
  IF( (opt ==1) or (opt==2) or &
     (opt==3) or (opt==4) ) then